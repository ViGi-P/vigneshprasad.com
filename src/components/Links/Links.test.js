import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Links } from './'

it('renders links', () => {
  const wrapper = shallow(<Links/>)
  expect(toJson(wrapper)).toMatchSnapshot()
})
