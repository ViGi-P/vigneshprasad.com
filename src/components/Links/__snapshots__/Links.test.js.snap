// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders links 1`] = `
<Row
  gutter={0}
>
  <Col
    className="Special"
    lg={12}
    md={10}
    sm={10}
    xs={24}
  >
    <h2>
      Frontend Developer specialising in React & React Native.
    </h2>
    <p
      className="Profile-links"
    >
      <a
        href="mailto:vigneshdmanfb@gmail.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Icon
          type="mail"
        />
      </a>
      <a
        href="https://github.com/ViGi-P"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Icon
          type="github"
        />
      </a>
      <a
        href="https://gitlab.com/ViGi-P"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Icon
          type="gitlab"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/vignesh-prasad"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Icon
          type="linkedin"
        />
      </a>
      <a
        href="https://profiles.udacity.com/p/11091209444"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Icon
          type="profile"
        />
      </a>
      <a
        href="https://twitter.com/vigneshdman"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Icon
          type="twitter"
        />
      </a>
    </p>
  </Col>
</Row>
`;
